jax/numpy/lax_numpy.py

```diff
@@ -1347,7 +1349,10 @@ def array(object, dtype=None, copy=True, order="K", ndmin=0):
       return out
   else:
     raise TypeError("Unexpected input type for array: {}".format(type(object)))
-asarray = array
+
+@_wraps(onp.asarray)
+def asarray(a, dtype=None, order=None):
+  return array(a, dtype=dtype, order=order)
```

Comment:
The NumPy implementation is

```
def asarray(a, dtype=None, order=None):
  return array(a, dtype, copy=False, order=order)
```

I think we should also use `copy=False` here.

Reply:
Done.